lamindb/db/_ingest.py

```diff
@@ -382,54 +360,77 @@ def _register_dobjects(self, jupynb_id, jupynb_v, jupynb_name):
                 log_file_name = str(filepath.relative_to(self._run.outdir))
             except ValueError:
                 log_file_name = filepath.name
-            self._ingestion_logs.append(
+            self._logs.append(
                 [
                     f"{log_file_name} ({dobject_id}, {dobject_v})",
                     f"{self._run.run_name} ({self._run.run_id})",
                     f"{settings.user.handle} ({settings.user.id})",
                 ]
             )
 
-    def _register_samples(self):
-        """Register entries in the techsample, biosample, biometa tables."""
-        sample_id = self._run.sample_id
-        if sample_id is None:
-            # insert techsample, biosample, and biometa entries
+    def _ingest_samples(self):
```

Comment:
@sunnyosun I would review the logic here carefully to make sure I correctly understood what we discussed.